nservicebus/errors/automatic-retries.md

```diff
@@ -70,6 +70,10 @@ NOTE: Retrying messages for extended periods of time would hide failures from op
 
 SLR can be configured in several ways.
 
+### Transport transaction requirements
+
+The SLR mechanism is implemented by rolling back the [transport transaction](/nservicebus/messaging/transactions.md) and scheduling message for [delayed-delivery](/nservicebus/messaging/delayed-delivery.md) next time it is picked from the input queue. Aborting the receive operation when transactions are turned off would results in message loss. Therefore SLR cannot be used when transport transactions are disabled.
```

Comment:
IMO it's saying something else. The message is scheduled for delayed delivery right away, which also implies that it will be delivered again (I think that's clear from "delayed delivery"). 
"scheduling message for delayed-delivery the next time it is picked from the input queue" sounds like the scheduling will happen when the message is picked for the second time, but why would it be picked again if it wasn't scheduled for delayed delivery?

If my understanding is accurate (first paragraph) then I'd change the text. I'll do a full review of both articles anyway, because they have other issues which predate your changes, so just let me know if I misunderstood something.